docs/source/using/installation.rst

```diff
@@ -1,70 +1,98 @@
+
+
+Enthought Deployment Manager
+----------------------------
+
+The BDSS, the Workflow Manager and plugins must be installed through the `Enthought Deployment
+Manager (EDM) <https://www.enthought.com/enthought-deployment-manager/>`_, a python environment and
+package manager similar to Anaconda. For new users it is worth examining EDM's
+`documentation <http://docs.enthought.com/edm/>`_.
 
-The last repository is optional, but recommended if you want to practice
-writing plugins.
+To install EDM, follow the instructions specific to your operating system
+`,here <https://docs.enthought.com/edm/installation.html>`_.
 
-If you never installed the Enthought Deployment Manager (EDM), perform the following operations
+Once EDM is installed enter the EDM shell,
 
 .. code-block:: console
 
-    wget https://package-data.enthought.com/edm/rh5_x86_64/1.11/edm_1.11.0_linux_x86_64.sh && bash ./edm_1.11.0_linux_x86_64.sh-b -f -p $HOME
-    export PATH=${HOME}/edm/bin:${PATH}
-    edm install --version 3.6 -y click setuptools
     edm shell
 
-If you instead already have an EDM installation and a default environment, perform the following
+and install the default ``edm`` environment,
 
 .. code-block:: console
 
-    edm shell
     edm install -y click setuptools
 
-Verify that your shell prompt now contains the string "``(edm)``".
-You are now in your default EDM environment, and we assume this environment to be the bootstrap environment.
-The BDSS software will not be installed in this environment, but in a separate one. The following
-commands however must be executed from the bootstrap environment.
+You are now in the ``edm`` environment and your shell prompt is prefixed
+with ``(edm)``.
+
+Repository Installation
+-----------------------
 
-Installation of the force BDSS runtime environment is performed with the
-following command. This should be done from the directory containing the ``force-bdss`` folder (named
-``Force-Project`` in this example)
+``edm`` is the 'bootstrap' environment, used to install the cloned respositories.
+For each respository in turn, cd into its directory and then install it with
+``python -m ci install``. i.e.,
 
 .. code-block:: console
 
-    ~/Force-Project (edm)$ cd force-bdss
-    ~/Force-Project/force-bdss (edm)$ python -m ci build-env
+    ~/Force-Project/force-bdss (edm)$ python -m ci install
+
+    ~/Force-Project/force-bdss (edm)$ cd ../force-wfmanager
+    ~/Force-Project/force-wfmanager (edm)$ python -m ci install
 
-This will create another edm environment called ``force-py36``.
-Do not enter this environment. 
+    ~/Force-Project/force-wfmanager (edm)$ cd ../force-bdss-plugin-enthought-example
+    ~/Force-Project/force-bdss-plugin-enthought-example (edm)$ python -m ci install
+
+    ~/Force-Project/force-wfmanager (edm)$ cd ../force-bdss-plugin-nevergrad
+    ~/Force-Project/force-bdss-plugin-nevergrad (edm)$ python -m ci install
+
+    ...etc
```

Comment:
These packages need to be installed inside the `force-py36` environment, so this section ("Repository Installation") should go after "The BDSS Runtime Environment"